packages/dashboard/src/components/dashboard.tsx

```diff
@@ -28,6 +29,28 @@ export interface SelectedFeature {
     runtimeArguments?: string;
 }
 
+interface IDashboardParams {
+    user_feature: string | undefined;
```

Comment:
is there a reason for the snake_case? we usually just using camel case.

Reply:
also, what is the user referring to here? maybe we can remove the prefix user_

Reply:
@barak007 let's put an action item to rethink the naming of it, 
it addresses the selected feature & config that is broken down in the dashboard (not the one that necessarily running as root feature by engineer)

Reply:
not sure why an action item is needed here,,, it's a simple thing that we could think about since it popped up in this PR.
If you want I will not insist to change this now...but I will ping you in a week to see what's going on with this.